react-function-components/src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const $root = document.getElementById('root');
+
+function CustomButton(props) {
+  return <button>{props.text}</button>;
+}
+
+const button = <CustomButton text="Click Me!"/>;
+
+ReactDOM.render(button, $root);
```

Comment:
You don't have to save it into a variable first, you can just directly pass a component to `render`:

```JSX
ReactDOM.render(<CustomButton />, $root);
```